bin/dry-run.rb

```diff
@@ -224,6 +224,10 @@
         [o.strip.downcase.to_sym, true]
       end
     end
+
+    $options[:updater_options].each do |name, val|
+      Dependabot::Experiment.register(name, val)
+    end
```

Comment:
Do we use `:updater_options` for more than experiments? Maybe it's not such a big deal for `dry-run`, but the name doesn't really fit the purpose.

Reply:
Yeah we use a few different names in different places, we ended up with an `options` arg for the `FileUpdater` with the idea that it could be used for more things than just experiments. I'm not sure that we ever have, and going forward we should probably rename things where it makes sense, but I was aiming to keep the change as small as possible